lib/application/workflow/cli/wikipedia_handler.ex

```diff
@@ -0,0 +1,36 @@
+defmodule WikipediaHandler do
+  def handle_wikipedia_article(uri) do
+    main_article = fetch_wikipedia_page(uri, 3)
```

Comment:
Should move the magic number (`3`, in this case) to another place (like `@main_article_attempts`)